### ISSUE TITLE
virtualization guest support darwin

### ISSUE THREAD

**Issue by clburlison** [chef/ohai#1163]
## Description

Currently, [virtualization.rb] on darwin only detects guest VMs in Parallels (I assume this works but I can't verify since I don't use Parallels). It would be great if support for VMWare Fusion & VirtualBox was added. I've used the following snippet in other deployments for detection purposes.

```ruby
      if node['hardware']['boot_rom_version'].include? 'VMW'
        virtual_type = 'vmware'
      elsif node['hardware']['boot_rom_version'].include? 'VirtualBox'
        virtual_type = 'virtualbox'
      else
```

I don't follow the ohai code base super closely but would a PR be accepted if I added two `elsif` blocks in the appropriate areas and included the above code? I don't know if this would cause an issue within Ohai since it would be depending on data from a separate plugin to be ran before hand.

## Ohai Version

13.8.0

## Platform Version

Darwin

## Ohai Output

When ran inside a VMWare Fusion VM:

```bash
Mac:~ username$ sudo ohai virtualization/systems
{

}
```

References:
* https://github.com/facebook/IT-CPE/pull/89/files
* https://github.com/munki/munki-facts/blob/master/facts/physical_or_virtual.py

[virtualization.rb]: https://github.com/chef/ohai/blob/3ffe07a3415a726a17b1bdc4626cabd32d986354/lib/ohai/plugins/darwin/virtualization.rb

**Comment by tas50** [comment]
Let me see if I can add vmware fusion. That should be pretty simple.

**Comment by tas50** [comment]
1 down:

{
  "systems": {
    "vmware": "guest"
  },
  "system": "vmware",
  "role": "guest"
}

**Comment by tas50** [comment]
Here's detection of both vmware and virtualbox. I tested it on a vmware fusion box I have. I don't have a virtualbox system, but I used the rom value in the facebook link.

https://github.com/chef/ohai/pull/1164

**Comment by thommay** [comment]
Fixed by #1164
